modules/smart-agent_logstash/conf/readme.yaml

```diff
@@ -0,0 +1,23 @@
+documentations:
+  - name: Smart Agent monitor
+    url: 'https://docs.signalfx.com/en/latest/integrations/agent/monitors/logstash.html'
+
+source_doc: |
+  Check the [integration
+  documentation](https://docs.signalfx.com/en/latest/integrations/agent/monitors/logstash.html)
+  in addition to the monitor one which it uses.
+
+  ### Monitors
+
+  You have to enable the following `extraGroups` in your monitor configuration:
+
+  * `pipeline`
```

Comment:
I would list extraMetrics instead of extraGroups to only enable what it is need by this module, the user still can enable the entire group if he wants to.

Reply:
Change made.